wiki/Tournaments/CWC/2019/en.md

```diff
@@ -0,0 +1,243 @@
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
+
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+   -  This may change with the actual Group Stage setup.
+
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| -----: | -------- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals - Week 1 | O & AB, AC, AD |
+| Finals - Week 2 | AE, P, Q |
+
+**Winning conditions:**
+   - In Group Stage, you need to win 5 maps to win a match. (Best-of-9)
+   - In the Round of 16 and the Quarterfinals, you need to win 6 maps to win a match. (Best-of-11)
+   - In the **Semifinals, Finals and Grand Finals**, you need to win 7 maps to win the match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'ScoreV2'. Room name must be "CWC 2019: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+
+2. Each team is free to select up to one warm-up map. Using beatmaps with questionable content is prohibited. All maps must be osu!catch specific maps.
+3. Each captain can ban one beatmap to be selected from the pool **at the start of the match**. These beatmaps are not allowed to be picked by any team in the entire match.  
+4. **After a team reaches 3 points the opposition can ban another map as a comeback mechanic.**. This means that both teams can potentially ban up to **two maps** during the entirety of the match.
+5. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
+6. Each captain must use "!roll" once in #multiplayer.
+   - **The winner of the !roll decides wheter they want to pick and ban first, or pick and ban second.**
+
+7. Captains may pick freely from any bracket.
+   - In case of a tie, the tiebreaker map must be played.
+
+8. Results of the Group Stage will be published via a Statistics sheet.
+
+### Mappool Instructions
+
+1. There will be 1 mappool for the Group Stage, 1 mappool for the Round of 16, 1 mappool for the Quarterfinals, 1 mappool for the Semifinals, 1 mappool for the Finals and 1 mappool for the Grand Finals.
+
```

Comment:
these empty spaces between the lists should be removed, it is giving the main list items more margin than the sub-lists